FWCore/Concurrency/interface/WaitingTaskWithArenaHolder.h

```diff
@@ -42,7 +42,7 @@ namespace edm {
 
     // Takes ownership of the underlying task and uses the current
     // arena.
-    explicit WaitingTaskWithArenaHolder(WaitingTaskHolder&& iTask);
+    explicit WaitingTaskWithArenaHolder(WaitingTaskHolder iTask);
```

Comment:
This change allows a `WaitingTaskHolder` to be "implicitly copied" into `WaitingTaskWithArenaHolder`, i.e.
```cpp
WaitingTaskHolder h;
WaitingTaskWithArenaHolder wta{h};
// instead of
WaitingTaskHolder h;
WaitingTaskWithArenaHolder wta{WaitingTaskHolder{h}};
```

The pattern of `WaitingTaskHolder` being moved should work as before.